models/tt_transformers/tt/model.py

```diff
@@ -317,6 +324,7 @@ def ttnn_prefill_forward(
         chunk_start_idx=None,
         get_last_token=-1,
         kv_cache=None,
+        **kwargs,
```

Comment:
This makes it hard to see which arguments could be expected. Why not list the potential arguments with default values above?